scripts/generate_website_docs.sh

```diff
@@ -4,8 +4,7 @@
 
 pip install -r docs-requirements.txt
 
-TMP_DIR=/tmp/website_docs
+TMP_DIR=/tmp/python_otel_docs
```

Comment:
can this script be removed altogether if the folder's no longer needed?

Reply:
As I mentioned in the opening comment I'm ...

> Keeping the script for now since it'll help with the rest of the https://github.com/open-telemetry/opentelemetry.io/issues/1109

Sorry, that parenthetical remark was easy to miss.

Once I'm done migrating all the docs, I'll remove the script. How does that sound?

Reply:
Thanks @chalin, yup I completely missed that 😄. Sounds good,  can you add an item to https://github.com/open-telemetry/opentelemetry.io/issues/1109 to that effect.

Reply:
@codeboten - yes, done.